tests/Api/Admin/OrdersTest.php

```diff
@@ -65,4 +67,240 @@ public function it_gets_orders_for_customer(): void
+
+        $this->client->request(
+            method: 'GET',
+            uri: '/api/v2/admin/addresses/' . $billingAddress->getId(),
+            server: $this->createFetchRequestHeaders('api@example.com'),
+        );
+
+        $this->assertResponse(
+            $this->client->getResponse(),
+            'admin/order/get_billing_address_of_placed_order_response',
+            Response::HTTP_OK,
+        );
+    }
+
+    /** @test */
+    public function it_updates_a_billing_address_of_placed_order(): void
+    {
+        $fixtures = $this->loadFixturesFromFiles([
+            'authentication/api_administrator.yaml',
+            'channel.yaml',
+            'order/customer.yaml',
+            'order/order.yaml',
+        ]);
+
+        /** @var AddressInterface $billingAddress */
+        $billingAddress = $fixtures['billing_address'];
+
+        $this->client->request(
+            method: 'PUT',
+            uri: '/api/v2/admin/addresses/' . $billingAddress->getId(),
+            server: $this->createUpdateRequestHeaders('api@example.com'),
+            content: json_encode([
+                'firstName' => 'Updated: Adam',
+                'lastName' => 'Updated: Handley',
+                'company'=> 'Updated: FMŻ',
+                'street' => 'Updated: Kościuszki 21',
+                'countryCode' => 'Updated: FR',
+                'city' => 'Updated: Bordeaux',
+                'postcode' => 'Updated: 99-999',
+                'phoneNumber' => 'Updated: 911213969',
+                'provinceCode' => 'Updated: PL-WP',
+                'provinceName' => 'Updated: wielkopolskie'
+            ]),
+        );
+
+        $this->assertResponse(
+            $this->client->getResponse(),
+            'admin/order/put_billing_address_of_placed_order_response',
+            Response::HTTP_OK,
+        );
+    }
+
+    /** @test */
+    public function it_prevents_customer_update_in_billing_address_of_placed_order(): void
+    {
+        $fixtures = $this->loadFixturesFromFiles([
+            'authentication/api_administrator.yaml',
+            'channel.yaml',
+            'order/order.yaml',
+            'order/customer.yaml',
+        ]);
+
+        /** @var AddressInterface $billingAddress */
+        $billingAddress = $fixtures['billing_address'];
+
+        /** @var CustomerInterface $customerTony */
+        $customerTony = $fixtures['customer_tony'];
+
+        /** @var CustomerInterface $customerDave */
+        $customerDave = $fixtures['customer_dave'];
+
+        $this->client->request(
+            method: 'PUT',
+            uri: '/api/v2/admin/addresses/' . $billingAddress->getId(),
+            server: $this->createFetchRequestHeaders('api@example.com'),
+            content: json_encode([
+                'customer' => '/api/v2/admin/customers/' . $customerDave->getId(),
+            ]),
+        );
+
+        $this->assertResponseCode($this->client->getResponse(), Response::HTTP_OK);
```

Comment:
Why don't you check the `customer` here?

Reply:
It seems you're correct. Since we have a serialized GET operation on the PUT response, we could omit the request below and directly check it here.